core/metacling/src/TClingBaseClassInfo.h

```diff
@@ -72,7 +72,7 @@ class TClingBaseClassInfo {
    bool          IsValid() const;
    int           Next();
    int           Next(int onlyDirect);
-   ptrdiff_t     Offset(void * address = 0, bool isDerivedObject = true) const;
+   intptr_t      Offset(void * address = 0, bool isDerivedObject = true) const;
```

Comment:
Why is this one needed? I thought `intptr_t` and `ptrdiff_t` would mostly be equivalent. Semantics?

Reply:
Yes, semantics. I use `intptr_t` almost everywhere

Reply:
Yes, but semantically here this is an `offset` and hence `ptrdiff_t`, isn't it?

Reply:
Right. Do you want me to change it?

Reply:
Yes, please :)

Reply:
OK

Reply:
>  result of pointer calculation?

When it is a difference, yes.  So the return value of the wrapper, 'fOffset` and `fIterStack` should be using `ptrdiff_t`